### DISCUSSION TITLE
How can I use strokeDasharray for bpmn2.Flow shapes?

### DISCUSSION THREAD

**Discussion by goyaljai1** [clientIO/joint#2823] [Q&A] [answered]
### Introduction

I am attempting to use `strokeDasharray` on `bpmn2.Flow` to create a dotted line effect, but the changes are not reflecting. Is this feature currently supported? Thank you!

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
You'd have to set the `borderStyle` to `null` first.

https://docs.jointjs.com/api/shapes/bpmn/#borderstyle

It's a special presentation attribute that sets `stroke-dasharray` based on its value and the current element size.
